### PR TITLE
Registry code cleanup for JMeter backend 

### PR DESCRIPTION
**Pull request by diegomarangoni**
EES-4629

This PR removes all unnecessary code after the complete migration to registry pattern related to JMeter backend.

### CODE REVIEW

**Review by boekkooi-fresh — COMMENTED** [review]
Why did you not just remove `pkg/backends/jmeter/jmeter.go`?

**Comment by diegomarangoni** [comment]
> Why did you not just remove `pkg/backends/jmeter/jmeter.go`?

I updated the PR

File: pkg/backends/jmeter/resources.go [hunk 1]
```diff
@@ -8,34 +8,35 @@ import (
...
+
+	"github.com/hellofresh/kangal/pkg/core/helper"
+	loadTestV1 "github.com/hellofresh/kangal/pkg/kubernetes/apis/loadtest/v1"
 )
 
 const (
-	// LoadTestLabel label used for JMeter resources
-	LoadTestLabel = "loadtest"
+	// loadTestLabel label used for JMeter resources
+	loadTestLabel = "loadtest"
```
Comment (boekkooi-fresh) [inline pkg/backends/jmeter/resources.go:23]: Please move the renames in this file to another PR. Or at least another commit.

File: pkg/backends/jmeter/backend_test.go [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Arbitrary test values
+	distributedPodsNum := int32(2)
+	namespace := "loadtest-namespace"
+	reportURL := "http://kangal-proxy.local/load-test/loadtest-name/report"
+
+	// Fake clients
+	logger, _ := zap.NewDevelopment()
```
Comment (vgarvardt) [inline pkg/backends/jmeter/backend_test.go:127]: zaptest pls

File: pkg/backends/jmeter/backend_test.go [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+		logger:          logger,
+		namespaceLister: namespaceLister,
+	}
+
+	err := b.Sync(ctx, loadTest, reportURL)
+	require.NoError(t, err, "Error when syncing")
+
+	services, err := kubeClient.CoreV1().Services(namespace).List(ctx, metaV1.ListOptions{})
+	require.NoError(t, err, "Error when listing services")
+	assert.NotZero(t, len(services.Items), "Expected non-zero service amount after CheckOrCreateResources but found zero services")
```
Comment (vgarvardt) [inline pkg/backends/jmeter/backend_test.go:167]:
```suggestion
	assert.NotEmpty(t, services.Items, "Expected non-zero service amount after CheckOrCreateResources but found zero services")
```

File: pkg/backends/jmeter/backend.go [hunk 4]
```diff
@@ -18,11 +20,29 @@ import (
 	clientSetV "github.com/hellofresh/kangal/pkg/kubernetes/generated/clientset/versioned"
 )
 
+var (
+	// MaxWaitTimeForPods is the time we should wait for worker pods to get to a "Running" state
+	MaxWaitTimeForPods = time.Minute * 10
+	// LoadTestWorkerLabelSelector is the selector used for selecting jmeter worker resources
+	LoadTestWorkerLabelSelector = fmt.Sprintf("%s=%s", loadTestWorkerPodLabelKey, loadTestWorkerPodLabelValue)
+	// ErrRequireMinOneDistributedPod Backend spec requires 1 or more DistributedPods
+	ErrRequireMinOneDistributedPod = errors.New("LoadTest must specify 1 or more DistributedPods")
```
[the remark this answers is not in the document]
Comment (diegomarangoni) [reply pkg/backends/jmeter/backend.go:29]: This is an existing error, changing here is out of scope and needs some discussion

**Comment by vgarvardt** [comment]
👍

**Pull request merged**